release-notes.txt

```diff
@@ -256,18 +256,18 @@ to all of you contributing in so many different ways to make RIOT worthwhile!
 
 More information
 ================
-http://www.riot-os.org
+https://www.riot-os.org
```

Comment:
Here and below: We shouldn't change old release notes.

Reply:
But old release notes are just documentation, does it really matters?

Reply:
Yes, because they should stay untouched for older releases IMHO. But maybe other maintainers have different opinions.